source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,46 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+}
+
+hx-def {
+  display: flex;
+
+  + hx-def {
+    margin-top: 1.25rem; // 20px
+  }
+}
+
+hx-dt {
+  #grid.span(2);
+  color: @gray-750;
+  display: block;
+  flex-shrink: 1;
+  font-weight: 400;
+  text-align: right;
+}
+
+hx-dd {
+  #grid.span(10);
+  display: block;
+  flex-shrink: 1;
+  margin-left: 1.25rem; // 20px
+}
+
+hx-dl.hxVertical {
+  hx-def {
+    flex-direction: column;
```

Comment:
instead of using flex-box maybe `display:block`?

Reply:
so I tried the display block, but the grid span mix-in we are using has `flex` widths and `max-widths`. Thus I used block, and the flex widths stayed on the element, but obviously aren't used. Then the `max-widths` would be the only thing applying. I rather not mix `display: flex;` and `display: block` in the component. So i'm sticking to the flex column that I had.